dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/OracleMetadataConnectorTest.java

```diff
@@ -43,8 +44,9 @@ public void addTasksTo_generatesMetadataSelectTasks() throws Exception {
     // Assert
     ImmutableList<String> selectSqls =
         tasks.stream()
-            .filter(task -> task instanceof SelectTask)
-            .map(task -> ((SelectTask) task).getSql())
+            .map(task -> GroupTask.getSql(task))
+            .filter(Optional::isPresent)
+            .map(Optional::get)
```

Comment:
I would suggest adding a method `optionalToStream` and then simplifying:
```
.flatMap(task -> optionalToStream(GroupTask.getSql(task)))
```

New method:
```
class OptionalUtils {
  ...
  private static <T> Stream<T> optionalToStream(Optional<T> optional) {
    return optional.map(Stream::of).orElseGet(Stream::empty);
  }
}
```

Reply:
Done, I added `optionalToStream`